### PR TITLE
create docs for HTMLTableColElement

### PR DESCRIPTION
**Pull request by fuchunhui**

### CODE REVIEW

**Comment by fuchunhui** [comment]
The link for `files/zh-cn/web/api/htmltablecolelement/valign/index.md` is https://github.com/mdn/translated-content/pull/23775

File: files/zh-cn/web/api/htmltablecolelement/span/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      <td>cell</td>
+      <td>cell</td>
+      <td>cell</td>
+      <td>cell</td>
+    </tr>
+  </tbody>
+</table>
+<button id="increase">增加列跨度</button>
+<button id="decrease">减少列跨度</button>
+<div>第一个 &lt;列&gt; 跨度 <output>2</output> 实际列。</div>
```
Comment (jasonren0403) [inline files/zh-cn/web/api/htmltablecolelement/span/index.md:50]:
```suggestion
<div>第一个 &lt;col&gt; 跨度 <output>2</output> 实际列。</div>
```

File: files/zh-cn/web/api/htmltablecolelement/span/index.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+decreaseButton.addEventListener("click", () => {
+  col.span = col.span - 1;
+
+  // 更新显示
+  output.textContent = col.span;
+});
+```
+
+### Result
```
Comment (jasonren0403) [inline files/zh-cn/web/api/htmltablecolelement/span/index.md:102]:
```suggestion
### 结果
```

File: files/zh-cn/web/api/htmltablecolelement/valign/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  sourceCommit: bb48907e64eb4bf60f17efd7d39b46c771d220a0
+---
+
+{{APIRef("HTML DOM")}}{{deprecated_header}}
+
+{{domxref("HTMLTableColElement")}} 接口的 **`vAlign`** 属性是一个指示如何在表格 {{htmlelement("col")}} 列元素中垂直对齐文本的字符串。
+
+> [!NOTE]
+> 此属性已弃用，应使用 CSS 在表格列中垂直对齐文本。使用 CSS {{cssxref("vertical-align")}} 属性，其用于垂直对齐列单元格中文本，且优先级更高。
+> 由于 {{htmlelement("td")}} 不是 {{htmlelement("col")}} 的子元素，因此不能直接在 {{HTMLElement("col")}} 上设置它，需要使用 `td:nth-child(n)` 或类似值（`n` 是列号）来选择列的单元格。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/htmltablecolelement/valign/index.md:14]:
```suggestion
> 此属性已弃用，应使用 CSS 在表格列中垂直对齐文本。使用 CSS {{cssxref("vertical-align")}} 属性，其用于垂直对齐列单元格中文本，且优先级更高。
> 
> 由于 {{htmlelement("td")}} 不是 {{htmlelement("col")}} 的子元素，因此不能直接在 {{HTMLElement("col")}} 上设置它，需要使用 `td:nth-child(n)` 或类似值（`n` 是列号）来选择列的单元格。
```

File: files/zh-cn/web/api/htmltablecolelement/valign/index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+可能的值有：`"top"`、`"middle"`、`"bottom"` 或 `"baseline"`。
+
+- `top`
+  - : 将文本与单元格顶部对齐。使用 `vertical-align: top` 代替。
+- `center`
+  - : 将文本与单元格垂直居中对齐，`middle` 的同义词。使用 `vertical-align: middle` 代替。
+- `middle`
+  - : 将文本与单元格垂直居中对齐。使用 `vertical-align: middle` 代替。
+- `bottom`
+  - : 将文本与单元格底部对齐。使用 `vertical-align:  bottom` 代替。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/htmltablecolelement/valign/index.md:27]:
```suggestion
  - : 将文本与单元格底部对齐。使用 `vertical-align: bottom` 代替。
```

**Review by jasonren0403 — COMMENTED** [review]
一些小问题

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**